cmd/compareBedDistanceBasedOnName/compareBedDistanceBasedOnName.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+)
+
+func compareBedDistanceBasedOnName(inputBed string, genomeBed string, outBed string) {
+	out := fileio.EasyCreate(outBed)
+	input := bed.Read(inputBed)
+	genome := bed.Read(genomeBed)
+
+	genomeMap := make(map[string][]bed.Bed)
+	var i int
+
+	for i = range genome {
+		genomeMap[genome[i].Name] = append(genomeMap[genome[i].Name], genome[i])
+	}
+
+	var inputNameField []bed.Bed
+	var found bool
+	var distance int
+	var lowestDistance int
+	var err error
+	var j int
+
+	for i := 0; i < len(input); i++ {
```

Comment:
This can be rewritten as follows:
for i := range input

both are functionally equivalent, but the range usage is stylistically preferred.